javascript-conditionals/main.js

```diff
@@ -0,0 +1,95 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } return false;
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } return false;
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } return false;
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } return false;
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } return false;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 8 && pH <= 14) {
+    return 'base';
+  } else if (pH <= 6 && pH >= 0) {
+    return 'acid';
+  } return 'invalid pH level';
+}
```

Comment:
what happens if `pH` equals `7.1` or `6.9`?

Reply:
In this case, it returned with the `invalid pH level`